shell/platform/windows/windows_lifecycle_manager.cc

```diff
@@ -163,4 +183,108 @@ void WindowsLifecycleManager::BeginProcessingClose() {
   process_close_ = true;
 }
 
+void WindowsLifecycleManager::SetLifecycleState(AppLifecycleState state) {
```

Comment:
Please add a `TODO` that we shouldn't send the message until we receive the system initialized message.